src/routes/SurveyFeedback/container.js

```diff
@@ -0,0 +1,97 @@
+import { Component } from 'preact';
+// import SDK from '../../api';
+import { Consumer } from '../../store';
+import SurveyFeedback from './component';
+// import { insert, createToken } from '../../components/helpers';
+import { Livechat } from '../../api';
+import { endChat } from '../../lib/room';
+import { createToken } from '../../components/helpers';
+
+export class SurveyFeedbackContainer extends Component {
+	handleSubmit = async(fields) => {
+
+		const { survey } = this.props;
+
+		const { alerts, dispatch, room: { _id: rid } } = this.props;
+
+		await dispatch({ loading: true });
+
+		try{
+			if (rid){
+				const data = survey.items.map(ratingFactorName => {
+					return {
+						name: ratingFactorName,
+						value: `${fields.rating}`
+					}
+				});
+				const packet = {
+					rid: rid,
+					data: data
+				}
+				Livechat.chatSurvey(packet).then(response => {
```

Comment:
Hello! I checked this PR and found an error that the comment message is not sent to the server. It depends on the server config for livechat.

Reply:
Hello @antkaz, U r correct. This PR doesn't send the comment message to the server. I tried sending it, but the server is not configured to handle it hence it was throwing an error. So in order to send the comment message, some of the interfaces and checks on the server will need to be modified.

Reply:
I can help you do it

Reply:
It would be awesome!

Reply:
I created new PR for ROcket.Chat https://github.com/RocketChat/Rocket.Chat/pull/18280 where I fixed old survey config and added feedback form config.
Also I created PR https://github.com/murtaza98/Rocket.Chat.Livechat/pull/2 where I fixed current code and added sending comment to Rocket.Chat server.
Feedback saves to room data 
![image](https://user-images.githubusercontent.com/19654261/87652897-394e8a80-c76e-11ea-8e6a-76ff193caa49.png)

Reply:
@renatobecker @murtaza98 can you review please